zng/docs/spec.md

```diff
@@ -511,15 +538,16 @@ An application-specific payload has the following form:
 Here, `<control code>` is a decimal integer in the range 6-127 and `<payload>`
 is any UTF-8 string with escaped newlines.
 
-### Type Grammar
+### 4.2 Type Grammar
 
 Given the above textual definitions and the underlying ZNG specification, a
 grammar describing the textual type encodings is:
 ```
 <stype> := bool | byte | int16 | uint16 | int32 | uint32 | int64 | uint64 | float64
-         | string | bytes | bstring | enum | ip | net | time | duration | null
+         | string | bytes | bstring | enum | ip | port | net | time | duration | null
```

Comment:
I noticed there's an entry for the `port` type in the section 3.1.1 Typedefs table, and it seems to be in the code, but it was't mentioned in this table here. I'm hoping someone who knows the code could plug in the correct suggestion. @alfred-landrum sounded like he's game to dig this up.